src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -410,8 +410,11 @@ private function calculateNumberOfColumns($rows)
     private function buildTableRows($rows)
     {
         $unmergedRows = array();
-        for ($rowKey = 0; $rowKey < count($rows); ++$rowKey) {
```

Comment:
The only change to do is to add a `\` in front of count IMHO.

Reply:
@okorneliuk as Nicolas said, `count()` is one of the few PHP functions that can be greatly optimized just by telling PHP that you are using the default function instead of your own: `\count(...)` is much faster to execute than `count(...)`. So please revert the proposed changes and add instead the `\` to `count()`. Thanks!